### DISCUSSION TITLE
How to lower step interval for dashboard metrics?

### DISCUSSION THREAD

**Discussion by ImVexed** [SigNoz/signoz#4135] [Q&A] [answered]
Hello! I'm using the rust example with the opentelemetry otlp exporter, and I'm trying to figure out how to get my graphs I'm making with the meters created in rust to step in an interval smaller than 60 seconds.

I've been scanning through the config files and don't really see anything. I see that in a panel itself the aggregation value is locked to 60 and can't be edited. Why is this? Should I just edit it in the json export and re-import it?

**Comment by srikanthccv** [comment, marked as answer]
Not being able to edit the step interval is a bug. It should be fixed. I want to point out that there will be a final limitation on how small step interval can be (in future if not implemented in some parts of the codebase already). Even if you edit and re-import it might not work because the interval is calculated dynamically based on the time range selected.

**Comment by amazingkj** [comment]
Hello! I also want to adjust the unit of aggregation at daily intervals, is there still no way? I want to count the volume by date on the dashboard.
